scripts/ci

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 set -e
 
+if [ ! -z ${1} ]; then
+    export CHART=${1}
+fi
+
 ./scripts/prepare
 ./scripts/generate-charts
 ./scripts/clean
```

Comment:
yeah, I think this is out of the scope of this PR. we can do clean up in the future. For now I would like to keep `scripts/ci` untouched so that it would not impact developer's normal workflow for testing. Beside `scripts/ci-wrapper` is only a wrapper for CI and doesn't intended to be run by developers.